web/components/manalink-card.tsx

```diff
@@ -82,75 +84,79 @@ export function ManalinkCardFromView(props: {
   const [details, setDetails] = useState(false)
```

Comment:
`details` doesn't sound like a boolean variable. Maybe `showDetails` and `setShowDetails`?